packages/hooks/src/useLandingCompanyDetails.ts

```diff
@@ -0,0 +1,27 @@
+import { useMemo } from 'react';
+import { useQuery } from '@deriv/api';
+
+type TUseLandingCompanyDetailsPayload = Parameters<typeof useQuery<'landing_company_details'>>['1']['payload'];
+
+/** A custom hook that returns the available landing companies of the user's country. */
+const useLandingCompanyDetails = (payload: TUseLandingCompanyDetailsPayload) => {
```

Comment:
Can move this to deriv api since this is BE API related. This should not cause an issue since this hook is not dependant on `useAuthorize`. Plus we dont need to pass payload as parameters here. Can refer to `useVerifyEmail` in api package. we pass the mutate down .

Reply:
@thisyahlen-deriv this API accepts payload to send the response, it doesn't mutate data. However I'll consider it moving to api if its applicable, Thanks for the suggestion